_targets.R

```diff
@@ -18,71 +19,73 @@ endDate <- "2015-05-01"
 #               multiple sites without making a target specified to each individually?
 p1_targets_list <- list(
   tar_target(
-    site_data_01427207_csv, 
+    site_data_01427207, 
     download_nwis_site_data(site_num = "01427207",
                             parameterCd = parameterCd,
                             startDate = startDate,
                             endDate = endDate,
-                            out_file = sprintf('1_fetch/out/nwis_%s_data.csv', '01427207')),
-    format = "file"
+                            out_file = sprintf('1_fetch/out/nwis_%s_data.csv', '01427207'))
```

Comment:
I think you want to delete these `out_file` arguments entirely since there is not a file being saved anymore

Reply:
Of course! thanks for catching!